packages/x-date-pickers-pro/src/SingleInputDateRangeField/tests/editing.SingleInputDateRangeField.test.tsx

```diff
@@ -19,7 +19,7 @@ describe('<SingleInputDateRangeField /> - Editing', () => {
               defaultValue={[adapter.date(), adapter.addYears(adapter.date(), 1)]}
             />,
           );
-          const input = screen.getByRole('textbox');
+          const input = screen.getByRole<HTMLInputElement>('textbox');
```

Comment:
Nitpick, I've created a util function `getTextbox` in order to avoid this duplicated typed getter everywhere. But in any case, it is also a perfectly viable solution. 👌 
P.S. Have you merged with the recent next? 🙈

Reply:
Fixed :+1: